addons/web/static/src/views/fields/field.js

```diff
@@ -162,33 +162,29 @@ export class Field extends Component {
 Field.template = "web.Field";
 
 Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
-    const name = node.getAttribute("name");
-    const widget = node.getAttribute("widget");
-    const fields = models[modelName];
-    const modifiers = JSON.parse(node.getAttribute("modifiers") || "{}");
-    const field = getFieldFromRegistry(fields[name].type, widget, viewType, jsClass);
     const fieldInfo = {
-        name,
         viewType,
-        widget,
-        modifiers,
-        field,
-        context: node.getAttribute("context") || "{}",
-        string: node.getAttribute("string") || fields[name].string,
-        help: node.getAttribute("help"),
-        onChange: archParseBoolean(node.getAttribute("on_change")),
-        forceSave: archParseBoolean(node.getAttribute("force_save")),
-        options: evaluateExpr(node.getAttribute("options") || "{}"),
-        alwaysInvisible: modifiers.invisible === true || modifiers.column_invisible === true,
-        decorations: {}, // populated below
-        attrs: {}, // populated below
+        modifiers: {},
+        context: "{}",
+        options: {},
+        alwaysInvisible: false,
+        decorations: {},
+        attrs: {},
     };
-    if (node.getAttribute("domain")) {
-        // TODO WOWl: remove with new model?
-        fieldInfo.domain = node.getAttribute("domain");
-    }
+
     for (const { name, value } of node.attributes) {
-        if (name.startsWith("decoration-")) {
+        if (["name", "widget", "context", "string", "help", "domain"].includes(name)) {
+            fieldInfo[name] = value;
+        } else if (["on_change", "force_save"].includes(name)) {
+            fieldInfo[name] = archParseBoolean(value);
```

Comment:
in fieldInfo, those ones are camelcased, so you need to handle them individually

Reply:
Indeed, fixed